jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/PathRequestContent.java

```diff
@@ -62,4 +71,20 @@ public String getContentType()
     {
         return contentType;
     }
+
+    /**
+     * @deprecated Use the {@link ByteBufferPool.Sized} in a constructor
+     */
+    @Deprecated (forRemoval = true, since = "12.0.11")
+    public void setUseDirectByteBuffers()
+    {
+    }
```

Comment:
This is now a change in behavior:

```java
var s = new PathContentSource(path);
s.setBufferSize(10 * 1024);
```

Updating Jetty results in the above code to compile, but not work as expected.